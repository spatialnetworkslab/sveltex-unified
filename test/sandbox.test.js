
import dedent from 'dedent'
import { processor } from '../src/index.js'

const process = md => {
  return processor()
    .processSync(md)
    .toString()
}

test('paragraphs within else-if blocks should be terminated', () => {
  const input = process(dedent`
    {#if a < 10}
    One
    {:else if a == 10}
    Two
    {:else}
    Three
    {/if}
  `)
  expect(input).toMatchSnapshot()
})
