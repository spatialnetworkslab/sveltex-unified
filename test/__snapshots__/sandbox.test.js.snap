// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components 1`] = `"<p>hello</p>"`;

exports[`paragraphs within else-if blocks should be terminated 1`] = `
"<p>{#if a &#x3C; 10}
One
{:else if a == 10}
Two
{:else}
Three
{/if}</p>"
`;
